backend/src/test/java/de/neuefische/backend/controller/LoginControllerTest.java

```diff
@@ -0,0 +1,146 @@
+//package de.neuefische.backend.controller;
```

Comment:
Bin mir etwas um unklaren warum eine komplett Auskommentierte Datei in den Push kommt?